stable/guardian/values.yaml

```diff
@@ -40,9 +62,17 @@ app:
     LOG_LEVEL: info
     AUTHENTICATED_USER_HEADER_KEY: x-authenticated-user-email
     NOTIFIER_PROVIDER: slack
-    JOBS_FETCH_RESOURCES_INTERVAL: "0 */2 * * *"
-    JOBS_REVOKE_EXPIRED_ACCESS_INTERVAL: "*/20 * * * *"
-    JOBS_EXPIRING_ACCESS_NOTIFICATION_INTERVAL: "0 9 * * *"
+    JOBS:
```

Comment:
Whats the reason that we need these configs separately mentioned from cron?

Reply:
These jobs are running inside the guardian server until now. We can enable and disable them. If disabled - we need to run as part of cron. 
For backward compatibility, the guardian server has everything enabled by default in the code. (we have to change this behavior in later versions - @rahmatrhd ). So unless one explicitly disabled them through server config, they will run inside the server too.

Reply:
I see, so in case it is still supported from guardian code side, we can keep it.
Otherwise I think we should remove it.

Plus I am also thinking if this works well with our `app` chart, as we have been using configs as KV pairs till now, but this seems to have a more nested structure.

Can you confirm if this has been tested ?

Reply:
I don't think it might work, will test this specific chart version in lower env and update here before merging.

Reply:
@rohilsurana @rahmatrhd 

The latest changes work. Had to fix issues with the app cron chart, and also used a flattened structure for the job keys in config

```shell
> k get cronjobs | grep "guardian"
guardian-appeal-expiration-reminder     0 9 * * *      False     0        <none>          4m27s
guardian-appeal-expiration-revocation   */20 * * * *   False     0        3m7s            4m27s
guardian-fetch-resources                0 */2 * * *    False     0        3m7s            4m27s
```

```shell
> k get pods | grep "fetch-resources"
guardian-fetch-resources-27670440-kzs67                0/1     Completed   0          101s
```